WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -848,6 +911,13 @@ private extension ProductSelectorView.Configuration {
 }
 
 private extension ProductsSection {
+    enum Localization {
+        static let scanProductRowTitle = NSLocalizedString(
+            "orderForm.products.add.scan.row.title",
+            value: "Scan product",
```

Comment:
This should be title case: pecCkj-eD-p2
```suggestion
            value: "Scan Product",
```

Reply:
Fixed: https://github.com/woocommerce/woocommerce-ios/pull/12033/commits/910e4147e0902b14bcb0fda98831e12916c95b8f